docs/ssl-setup.rst

```diff
@@ -0,0 +1,20 @@
+=================
+Configure SSL/TLS
```

Comment:
Can you add a line in the README or index of setup in the docs as a link to this?

Reply:
After this PR and after we update Sphinx docs it would be in the left side menu of our Sphinx docs. Would you still like me to reference it in README and/or the setup docs?